testsuite/features/step_definitions/command_steps.rb

```diff
@@ -383,6 +383,19 @@
   end
 end
 
+When(/^I increase Java log level to "([^"]*)" on server$/) do |log_level|
+  $server.run(
+    "echo -e >>/srv/tomcat/webapps/rhn/WEB-INF/classes/log4j.properties << CONFIG
+log4j.logger.com.suse.manager.reactor=#{log_level.upcase}
+log4j.logger.com.suse.manager.webui.services.impl=#{log_level.upcase}
+CONFIG"
+  )
+end
+
+And(/^I restart the tomcat service$/) do
```

Comment:
`When`

Reply:
Tbh this is the most confusing part since I could find step implementations for both `When` and `And`, but I think they work the same. I'll change it to `When`, just wondered if you have a policy around that

Reply:
Yes, it works the same. Yes there's a policvy. `And ` is a shortcut for "same as previous line" in the feature test (in this case `When`), it has no reason to appear in the step definitions.